src/annotator/sidebar.js

```diff
@@ -48,48 +44,58 @@ function createSidebarIframe(config) {
   return sidebarFrame;
 }
 
+const defaultConfig = {};
+
 /**
  * The `Sidebar` class creates the sidebar application iframe and its container,
  * as well as the adjacent controls.
  */
 export default class Sidebar extends Guest {
+  /**
+   * @param {HTMLElement} element
+   * @param {Record<string, any>} config
+   */
   constructor(element, config) {
-    if (config.theme === 'clean' || config.externalContainerSelector) {
-      delete config.pluginClasses.BucketBar;
-    }
-
-    let externalContainer = null;
-
-    if (config.externalContainerSelector) {
-      externalContainer = document.querySelector(
-        config.externalContainerSelector
-      );
-    }
+    super(element, { ...defaultConfig, ...config });
```

Comment:
`defaultConfig` is empty, so I believe this is equivalent to `super(element, config)`.

Reply:
Yes, you are correct.

At some point I removed the `defaultConfig`, I added back to make it consistent to the `pdf-sidebar.js` class. I thought maybe in the future we want to add configuration at this level.

I am happy to remove it if you wish.

Reply:
You are correct.

At some point I remove the `defaultConfig`, then I put it back to make if consistent with the pdf-sidebar.js. I thought that maybe in the future it could be handy.

I am happy to remove it entirely.